src/utils.js

```diff
@@ -1,9 +1,11 @@
 // @flow
-import { memoize } from 'lodash'
-import type { SpacingProps, Noop } from './types'
+import { memoize, isString, isFinite, flow } from 'lodash'
```

Comment:
All this methods will increase the bundle size, any time we can do without them in reflex we should. It's a bit of a balancing act between what's already in use and how much value new ones add